torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -2273,47 +2306,50 @@ def create_wrapped_node(
     ):
         from .._trace_wrapped_higher_order_op import TransformGetItemToIndex
 
-        tx: InstructionTranslator = tx
+        @_make_inlinable
+        def _create_scalar(query):
+            return query.new_empty([], dtype=torch.int32)
 
-        def create_scalar():
-            return query.call_method(
-                tx,
-                "new_empty",
-                (VariableTracker.build(tx, []),),
-                {
-                    "dtype": VariableTracker.build(tx, torch.int32),
-                },
-            )
+        if fn_name == "score_mod":
 
-        with discard_graph_changes(tx):
-            bhmn = [create_scalar() for _ in range(4)]
-            if fn_name == "score_mod":
-                scores_require_grad: bool = query.requires_grad
-                score = query.call_method(
-                    tx,
-                    "new_empty",
-                    (VariableTracker.build(tx, []),),
-                    {"requires_grad": VariableTracker.build(tx, scores_require_grad)},
+            @_make_inlinable
```

Comment:
The other alternative of using _make_inlinable is to define the function in a file that's not skipped.